src/containers/NavbarContainer.js

```diff
@@ -0,0 +1,19 @@
+import {connect} from 'react-redux';
+import Navbar from '../components/navbar'
+import {readConfigFile} from '../utils/config'
+import {loadConfig} from '../actions/config'
+
+function mapStateToProps(state, ownProps) {
+    return {}
```

Comment:
Should it return an empty body? If it's not the final version I suggest a //TODO comment.

Reply:
I It just example if anyone would want to add props to a component